cloud2sql/arrow/__init__.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass
+from typing import Union, Literal, final
+from pathlib import Path
+
+
+@final
+@dataclass(frozen=True)
+class S3Bucket:
+    region: str
```

Comment:
why region? aren't s3 buckets global?

Reply:
if you don't specify it, then it will default to something. better give users a choice